src/tests/functionality/test_sessions.py

```diff
@@ -0,0 +1,32 @@
+def test_main_page(test_client):
```

Comment:
You've done - can be really cool to add a short CI to run those tests - will strengthen GitHub actions

Reply:
I'll add a CI